pgjdbc/src/main/java/org/postgresql/Driver.java

```diff
@@ -477,8 +482,11 @@ public int getMajorVersion()
         return MAJORVERSION;
     }
 
-
-    public static final int MINORVERSION = @MINORVERSION@;
+    public static final int MINORVERSION =
```

Comment:
```
//#if false
...
//#endif
```
turned out to play bad with "source autoformat", thus I went with `+";"$*//*-*/` stuff